eng/pipelines/common/platform-matrix.yml

```diff
@@ -190,15 +190,9 @@ jobs:
       platform: FreeBSD_x64
       container:
         image: ubuntu-18.04-cross-freebsd-11-20200403161435-3d73ec8
-        registry: mcr
       jobParameters:
-        runtimeFlavor: ${{ parameters.runtimeFlavor }}
         stagedBuild: ${{ parameters.stagedBuild }}
         buildConfig: ${{ parameters.buildConfig }}
-        ${{ if eq(parameters.passPlatforms, true) }}:
-          platforms: ${{ parameters.platforms }}
-        helixQueueGroup: ${{ parameters.helixQueueGroup }}
```

Comment:
Why are we removing `helixQueueGroup`?

Reply:
thais was based on your previous comment. I guess you meant just only the last line of each block - e.g. not the whole section showing, right? I'll give it another try when we have update docker image. I got delayed because of Alpine breakage.

Reply:
Yeah, I meant removing: stagedBuild and managedTestBuildOsGroup only.

Reply:
added back what I removed by mistake. We will not have  Helix runs right now, but I think it would be nice to keep it ready.